### PR TITLE
Remove VM random tests that contain `BALANCE`

### PR DESCRIPTION
**Pull request by pirapira**
because these require special execution mode for VM tests.

See https://github.com/ethereum/tests/issues/415

### CODE REVIEW

File: VMTests/vmRandomTest/201503110226PYTHON_DUP6.json [hunk 1]
```diff
@@ -1,51 +0,0 @@
-{
-    "201503110226PYTHON_DUP6" : {
```
Comment (winsvega) [inline VMTests/vmRandomTest/201503110226PYTHON_DUP6.json:2]: this test was failing on python someday

Comment (pirapira) [reply VMTests/vmRandomTest/201503110226PYTHON_DUP6.json:2]: Is that a reason to keep the test or to remove the test?

Comment (winsvega) [reply VMTests/vmRandomTest/201503110226PYTHON_DUP6.json:2]: was a long time ago. but if it contains CALL CREATE SELFDESTRUCT then it probably does not make sense to keep it in VMTests

Comment (pirapira) [reply VMTests/vmRandomTest/201503110226PYTHON_DUP6.json:2]: It contains `BALANCE` so it's being removed.

Comment (winsvega) [reply VMTests/vmRandomTest/201503110226PYTHON_DUP6.json:2]: the balance is easy to replace with any other opcode or remove it. maybe the issue was in other part of the code.

Comment (pirapira) [reply VMTests/vmRandomTest/201503110226PYTHON_DUP6.json:2]: It doesn't matter if `BALANCE` is easy or not.  When a client is running a VM test, the client should contain no world state, so the client should not be able to run BALANCE.

Comment (pirapira) [reply VMTests/vmRandomTest/201503110226PYTHON_DUP6.json:2]: So, I'll move this test to GeneralStateTest.

Comment (winsvega) [reply VMTests/vmRandomTest/201503110226PYTHON_DUP6.json:2]: ok

Comment (pirapira) [reply VMTests/vmRandomTest/201503110226PYTHON_DUP6.json:2]: I added this one as a generalstate test in this PR.

**Comment by pirapira** [comment]
I translated the test into a GeneralStateTest.

File: src/GeneralStateTestsFiller/stBugs/201503110226PYTHON_DUP6Filler.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        },
+        "expect" : [
+            {
+	            "indexes" : {
+	                "data" : -1,
+	                "gas" : -1,
+	                "value" : -1
+	            },
+	            "network" : [">=Frontier"],
+                "result" : {
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stBugs/201503110226PYTHON_DUP6Filler.json]: nothing really have changed here. perhaps better put gas consumed or nonce change into the expect section ?

Comment (pirapira) [reply src/GeneralStateTestsFiller/stBugs/201503110226PYTHON_DUP6Filler.json]: I wanted to change nothing, but `BALANCE` cannot live in VMtests, so the test moved.  It just moved without any additional guarantees.

Comment (winsvega) [reply src/GeneralStateTestsFiller/stBugs/201503110226PYTHON_DUP6Filler.json]: could be poot into random tests then

Comment (pirapira) [reply src/GeneralStateTestsFiller/stBugs/201503110226PYTHON_DUP6Filler.json]: I'll do that.

**Comment by pirapira** [comment]
@winsvega I moved the test into `stRandom2`.

**Pull request merged**